test-app/package.json

```diff
@@ -0,0 +1,107 @@
+{
+  "name": "test-app",
+  "version": "7.0.1",
+  "description": "An Ember.js addon for the painless support of keyboard events",
+  "keywords": [],
+  "repository": "https://github.com/adopted-ember-addons/ember-keyboard",
+  "license": "MIT",
+  "author": "",
+  "directories": {
+    "doc": "doc",
+    "test": "tests"
+  },
+  "scripts": {
+    "build": "ember build --environment=production",
+    "deploy": "yarn ember github-pages:commit --message \"Deploy gh-pages from commit $(git rev-parse HEAD)\"; git push; git checkout -",
```

Comment:
Will this still work with the changed paths?

Reply:
@lukemelia I don't see why it wouldn't work:
`ember-cli-github-pages` is dependency of the `test-app` hence all should just work.

The only difference if that `yarn deploy` command need to be run from `/test-app` folder instead of root folder.

unless I'm missing something

Reply:
Let's try it after merging and find out!